.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Please consider the following when opening a pull request:
+
+- Link to the relevant JIRA ticket(s) and tag any relevant team(s).
+- Squash your changes down into one or more discrete commits.
+  In each commit, include description that could help a developer
+  several months from now.
+- If running `make upgrade`, run _as close to the time of merging as possible_
```

Comment:
How about also adding something like:
Put the output of `make upgrade` in its own separate commit, decoupled from other code changes.